openapi.yaml

```diff
@@ -3062,6 +3062,857 @@ paths:
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              -X POST https://api.linode.com/v4beta/databases/mysql/instances/123/backups/456/restore
+      - lang: CLI
+        source: >
+          linode-cli databases mysql-backup-restore 123 456
+  /databases/mysql/instances/{instanceId}/credentials:
+    x-linode-cli-command: databases
+    parameters:
+      - name: instanceId
+        in: path
+        description: The ID of the Managed MySQL Database.
+        required: true
+        schema:
+          type: integer
+    get:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Credentials View
+      operationId: getDatabasesMySQLInstanceCredentials
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-creds-view
+      x-linode-grant: unrestricted only
+      description: |
+        Display the the root username and password for a Managed MySQL Database.
+
+        The Database must have an `active` status to perform this command.
+
+        Only unrestricted Users can access this command.
+
+        **Beta**: This endpoint is in **closed** beta. To get early access and test new Linode products before they
+        hit the market, join the [Linode Green Light](https://www.linode.com/green-light/) program.
+        When accessing this endpoint, please make sure to prepend all requests with `/v4beta` instead of `/v4`, and be
+        aware that this endpoint may receive breaking updates in the future. This notice will be removed when this out of
+        beta.
+      security:
+      - personalAccessToken: []
+      responses:
+        '200':
+          description: Managed Database root username and password.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DatabaseCredentials'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4beta/databases/mysql/instances/123/credentials/
+      - lang: CLI
+        source: >
+          linode-cli databases mysql-creds-view 123
+  /databases/mysql/instances/{instanceId}/credentials/reset:
+    x-linode-cli-command: databases
+    parameters:
+      - name: instanceId
+        in: path
+        description: The ID of the Managed MySQL Database.
+        required: true
+        schema:
+          type: integer
+    post:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Credentials Reset
+      operationId: postDatabasesMySQLInstanceCredentialsReset
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-creds-reset
+      x-linode-grant: unrestricted only
+      description: |
+        Reset the root password for a Managed MySQL Database.
```

Comment:
We may want to add a note here that it takes a few seconds for the credentials to reset and show the new creds.